ddtrace/debugging/_debugger.py

```diff
@@ -283,6 +288,22 @@ def __init__(self, tracer: Optional[Tracer] = None) -> None:
 
         log.debug("%s initialized (service name: %s)", self.__class__.__name__, service_name)
 
+    def _install_file_probes(self, path: str) -> None:
+        try:
+            with open(path, "r") as f:
+                raw_probes = json.load(f)
```

Comment:
If streaming is not required we could
```suggestion
            raw_probes = json.loads(path.read_text())
```
otherwise we can open the file with `path.open() as f`